client/app/cloud/project/compute/infrastructure/virtualMachine/delete/cloud-project-compute-infrastructure-virtual-machine-delete.controller.js

```diff
@@ -1,14 +1,12 @@
 "use strict";
 
 angular.module("managerApp")
-  .controller("CloudprojectcomputeinfrastructurevirtualmachinedeleteCtrl", function ($uibModalInstance, $stateParams, params, OvhApiCloudProjectIpFailover) {
-        var self = this,
-            serviceName = $stateParams.projectId;
-
-        var vmToDelete = params;
+    .controller("CloudprojectcomputeinfrastructurevirtualmachinedeleteCtrl", function ($uibModalInstance, params) {
```

Comment:
Could be written as a `class`? 💄

Reply:
It's a hotfix 😄. I'm all about refactoring to classes but not in a hotfix